### ISSUE TITLE
The package react-native@0.14.0 does not satisfy its siblings' peerDependencies

### ISSUE THREAD

**Issue by bartonhammond** [facebook/react-native-fbsdk#62]
Peer react-native-fbsdkcore@0.0.6 wants react-native@^v0.13.0

I tried changing package.json to force 0.13.0 but then project would not compile in XCode

**Comment by dzhuowen** [comment]
Can you provide more detail? Because ^0.13.0 should accept 0.14.0 as well. http://stackoverflow.com/questions/22343224/difference-between-tilde-and-caret-in-package-json

**Comment by philikon** [comment]
0.x is treated differently by semver.

**Comment by bartonhammond** [comment]
My steps:
1) react-native init foo & cd foo
2) npm install --save react-native-fbsdkcore
```
npm WARN peerDependencies The peer dependency react-native@^v0.13.0 included from react-native-fbsdkcore will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Darwin 14.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "--save" "react-native-fbsdkcore"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react-native@0.14.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-native-fbsdkcore@0.0.6 wants react-native@^v0.13.0

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/barton/projects/myFamilyVoice/foo/npm-debug.log

```


i am on node 4.2.1

**Comment by HarrisRobin** [comment]
+1

**Comment by voxmachina** [comment]
+1

**Comment by pranny** [comment]
+1 Getting the following WARNING

```
$ npm install --save react-native-fbsdklogin
AwesomeApp@0.0.1 /Users/developer/workspaceAwesomeApp
├── UNMET PEER DEPENDENCY react-native@0.14.1
└── react-native-fbsdklogin@0.0.6

npm WARN EPEERINVALID react-native-fbsdklogin@0.0.6 requires a peer of react-native@^v0.13.0 but none was installed.
```

**Comment by dzhuowen** [comment]
Thanks @philikon for pointing out the Magic Zero issue. ^0.13.0 doesn't include 0.14.0 and node is complaining about this.
To avoid this problem for now, please skip the section "JavaScript packages" and git clone this repository. Then copy react-native-fbsdkxxx/ into directory <project name>/node_modules.

**Comment by antigirl** [comment]
:sob:

**Comment by grabbou** [comment]
Unfortunately it does not work, can be tested here -> http://semver.npmjs.com/

Changing to `>=v0.13.0` will do the trick. Any chances it gets fixed soon? Need to roll out a new update and this kinda stops me from it :)

**Comment by dzhuowen** [comment]
Solved by pull request #66

**Comment by navels** [comment]
I'm getting this error again with react-native 0.18.0-rc, which resolves when I change the version in react-native/package.json to 0.18.0.
